pkg/client/rekor_client.go

```diff
@@ -32,7 +33,14 @@ func GetRekorClient(rekorServerURL string, opts ...Option) (*client.Rekor, error
 	}
 	o := makeOptions(opts...)
 
-	rt := httptransport.New(url.Host, client.DefaultBasePath, []string{url.Scheme})
+	retryableClient := retryablehttp.NewClient()
```

Comment:
was user-agent every set from the options?

Reply:
It’s set on line 41 based on what the caller passes in.

Reply:
OHH thank you! i was double-checking this for a fulcio retry client. thank you!